### PR TITLE
Updated content for Alerting feature

### PR DESCRIPTION
**Pull request by dsamojlenko**
# Summary | Résumé

Updates all the content on the Alerting feature, and a few other minor fixes/changes along the way.

### CODE REVIEW

File: locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -804,49 +800,45 @@ msgstr "Adresse"
...
-msgstr "En cliquant sur « Notifier les visiteurs de ce lieu », vous confirmez que ces renseignements sont exacts et complets selon vous."
+msgid ""
+"\n"
+"        <p>When you “Send QR alert”,  you confirm you’ve correctly entered all details from the health authority.</p>\n"
+"        <p>It’s not possible to cancel an alert after sending.</p>\n"
+"      "
+msgstr ""
+"\n"
+"        <p>En cliquant sur « Notifier les visiteurs de ce lieu », vous confirmez que ces renseignements sont exacts et complets selon vous.</p>\n"
+"        <p>It’s not possible to cancel an alert after sending.</p>\n"
```
Comment (dsamojlenko) [inline locale/fr/LC_MESSAGES/django.po:828]: @EliseKa svp

File: locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -804,49 +801,45 @@ msgstr "Adresse"
...
 #: outbreaks/templates/confirm.html
 msgid "severity"
 msgstr "Gravité"
 
 #: outbreaks/templates/confirm.html
-msgid "By selecting “Notify people who were there”,  you confirm you believe this information is correct and complete."
-msgstr "En cliquant sur « Notifier les visiteurs de ce lieu », vous confirmez que ces renseignements sont exacts et complets selon vous."
+msgid ""
+"\n"
+"        <p>When you “Send QR alert”,  you confirm you’ve correctly entered all details from the health authority.</p>\n"
```
Comment (EliseKa) [inline locale/fr/LC_MESSAGES/django.po:823]:
```suggestion
"        <p>When you “Send alert”,  you confirm you’ve correctly entered all details of the possible exposure.</p>\n"
```

File: locale/fr/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -804,49 +801,45 @@ msgstr "Adresse"
...
+"        <p>It’s not possible to cancel an alert after sending.</p>\n"
+"      "
+msgstr ""
+"\n"
+"        <p>En cliquant sur « Notifier les visiteurs de ce lieu », vous confirmez que ces renseignements sont exacts et complets selon vous.</p>\n"
```
Comment (EliseKa) [inline locale/fr/LC_MESSAGES/django.po:828]:
```suggestion
"        <p>En cliquant sur « Envoyer l’alerte », vous confirmez avoir correctement entré les détails de l’exposition potentielle.</p>\n"
```

File: locale/fr/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -804,49 +801,45 @@ msgstr "Adresse"
...
-msgstr "Notifier les visiteurs de ce lieu"
-
-#: outbreaks/templates/confirmed.html
-msgid "Notification sent"
-msgstr "Notification envoyée"
-
-#: outbreaks/templates/confirmed.html
-msgid "You’ve notified people"
-msgstr "Notification envoyée"
+msgid "Send QR alert"
```
Comment (EliseKa) [inline locale/fr/LC_MESSAGES/django.po:833]:
```suggestion
msgid "Send alert"
```

File: locale/fr/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -804,49 +801,45 @@ msgstr "Adresse"
...
+msgstr "Envoyer l’alerte QR"
```
Comment (EliseKa) [inline locale/fr/LC_MESSAGES/django.po:834]:
```suggestion
msgstr "Envoyer l’alerte"
```

File: locale/fr/LC_MESSAGES/django.po (hunk 1 above)
Comment (EliseKa) [reply locale/fr/LC_MESSAGES/django.po:829]:
```suggestion
"        <p>Notez qu’une fois envoyée, l’alerte ne peut pas être annulée.</p>\n"
```

**Review by EliseKa — COMMENTED** [review]
Updated a few lines to latest version.

File: outbreaks/forms.py [hunk 6]
```diff
@@ -97,7 +97,7 @@ def validate_date_entry(form, start_date, end_date, alert_location):
         tz = pytz.timezone(settings.PORTAL_LOCAL_TZ or settings.TIME_ZONE or "UTC")
         start_later_end_error_msg = _('"To" must be later than "From".')
         overlap_notification_error_tmpl = _(
-            "Someone already notified people who were there between {} and {}."
+            "Your team already alerted people who scanned the QR code on {}."
```
Comment (dorianjp) [inline outbreaks/forms.py:100]: I know this is in the figma, but I wonder if they considered overlap which has nothing to do with the start time. If an outbreak started at 8am-4pm, and they enter 3pm-11pm for the next, they would need to go lookup history to get more info. Thoughts @ioanacontu ?

File: locale/fr/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -1044,18 +1045,30 @@ msgstr "Adresse courriel"
...
+msgid "Alert people who scanned a QR code and may have been exposed during their visit."
+msgstr "Les personnes qui ont scanné le code QR de ce lieu ou de cet événement seront alertées d’un risque d’exposition."
 
 #: outbreaks/templates/search.html
 msgid "Enter at least 3 characters"
 msgstr "Saisissez au moins 3 caractères"
 
+#: outbreaks/templates/search.html
+msgid "results in your region for"
+msgstr ""
```
[the remark this answers is not in the document]
Comment (EliseKa) [reply locale/fr/LC_MESSAGES/django.po:1070]:
```suggestion
msgstr "résultats dans votre région pour"
```

File: locale/fr/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -1069,7 +1082,7 @@ msgid "location named "
 msgstr ""
 
 #: outbreaks/templates/search.html
-msgid "No results for"
+msgid "No results in your region for"
 msgstr "Aucun résultat pour"
```
[the remark this answers is not in the document]
Comment (EliseKa) [reply locale/fr/LC_MESSAGES/django.po:1086]:
```suggestion
msgstr "Aucun résultat dans votre région pour"
```

**Review by dorianjp — APPROVED** [review]
lgtm

**Pull request merged**